app/models/like.rb

```diff
@@ -0,0 +1,2 @@
+class Like < ApplicationRecord
+end
```

Comment:
- According to the ERD diagram that was provided, `likes` belong to `users` and `posts`. Kindly set up a `belongs_to` association to handle this. 😉 

- Let's add a method that updates the likes counter for a post. 😄